pycognito/utils.py

```diff
@@ -0,0 +1,89 @@
+from enum import Enum
+
+import requests.auth
+import requests
+
+from . import Cognito
+
+
+class RequestsSrpAuth(requests.auth.AuthBase):
+    """
+    A Requests Auth Plugin to automatically populate Authorization header
+    with a Cognito token.
+
+    Example:
+
+    ```
+    import requests
+    from pycognito.utils import RequestsSrpAuth
+
+    auth = RequestsSrpAuth(
+        username='myusername',
+        password='secret',
+        user_pool_id='eu-west-1_1234567',
+        client_id='4dn6jbcbhqcofxyczo3ms9z4cc',
+        user_pool_region='eu-west-1',
+    )
+
+    response = requests.get('http://test.com', auth=auth)
+    ```
+    """
+
+    class TokenType(str, Enum):
+        ID_TOKEN = "id_token"
+        ACCESS_TOKEN = "access_token"
```

Comment:
Can you move that out of class on top of this file?

Reply:
Done